[Bug][CI][E2E] Fail to start Mysql Container

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

At present, some e2e cases failed because `mysql:latest` could not start.
This PR(https://github.com/docker-library/mysql/commit/ffa6423ca24168e4d96631b5e8f536ac826d2a5b) upgraded `mysql:latest` to 8.3 from 8.0, and caused seatunnel e2e test failures.

### SeaTunnel Version

dev

### SeaTunnel Config

```conf
no
```


### Running Command

```shell
no
```


### Error Exception

```log
2024-01-22 02:08:47,073 INFO  🐳 [mysql:latest] - Creating container for image: mysql:latest
2024-01-22 02:08:47,116 INFO  🐳 [mysql:latest] - Container mysql:latest is starting: 4be50f3ec49d15c8d229d555699896f3cbf053b6b7c435fa29c1126f10c83ab4
2024-01-22 02:08:47,322 INFO  🐳 [mysql:latest] - Waiting for database connection to become available at jdbc:mysql://localhost:3306/seatunnel using query 'SELECT 1'
2024-01-22 02:08:47,325 INFO  🐳 [mysql:latest] - STDOUT: 2024-01-22 02:08:47+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.3.0-1.el8 started.
2024-01-22 02:08:47,461 INFO  🐳 [mysql:latest] - STDOUT: 2024-01-22 02:08:47+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2024-01-22 02:08:47,469 INFO  🐳 [mysql:latest] - STDOUT: 2024-01-22 02:08:47+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.3.0-1.el8 started.
2024-01-22 02:08:47,595 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22 02:08:47+00:00 [Warn] [Entrypoint]: MYSQL_PASSWORD specified, but missing MYSQL_USER; MYSQL_PASSWORD will be ignored
2024-01-22 02:08:47,598 INFO  🐳 [mysql:latest] - STDOUT: 2024-01-22 02:08:47+00:00 [Note] [Entrypoint]: Initializing database files
2024-01-22 02:08:47,607 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:47.604514Z 0 [System] [MY-015017] [Server] MySQL Server Initialization - start.
2024-01-22 02:08:47,607 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:47.605828Z 0 [System] [MY-013169] [Server] /usr/sbin/mysqld (mysqld 8.3.0) initializing of server in progress as process 81
2024-01-22 02:08:47,609 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:47.609162Z 0 [Warning] [MY-013907] [InnoDB] Deprecated configuration parameters innodb_log_file_size and/or innodb_log_files_in_group have been used to compute innodb_redo_log_capacity=10485760. Please use innodb_redo_log_capacity instead.
2024-01-22 02:08:47,610 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:47.610200Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
2024-01-22 02:08:47,818 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:47.817893Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.
2024-01-22 02:08:48,528 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:48.528100Z 0 [ERROR] [MY-000068] [Server] unknown option '--skip-host-cache'.
2024-01-22 02:08:48,528 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:48.528607Z 0 [ERROR] [MY-013236] [Server] The designated data directory /var/lib/mysql/ is unusable. You can remove all files that the server added to it.
2024-01-22 02:08:48,528 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:48.528624Z 0 [ERROR] [MY-010119] [Server] Aborting
2024-01-22 02:08:49,898 INFO  🐳 [mysql:latest] - STDERR: 2024-01-22T02:08:49.897926Z 0 [System] [MY-015018] [Server] MySQL Server Initialization - end.
2024-01-22 02:10:47,384 ERROR 🐳 [mysql:latest] - Could not start container
java.lang.IllegalStateException: Container exited with code 1
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:514) ~[testcontainers-1.17.6.jar:1.17.6]
	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:344) ~[testcontainers-1.17.6.jar:1.17.6]
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81) ~[duct-tape-1.0.8.jar:?]
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334) ~[testcontainers-1.17.6.jar:1.17.6]
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322) ~[testcontainers-1.17.6.jar:1.17.6]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:719) [?:1.8.0_392]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:701) [?:1.8.0_392]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:456) [?:1.8.0_392]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_392]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_392]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_392]
2024-01-22 02:10:47,406 ERROR 🐳 [mysql:latest] - Log output from the failed container:
```


### Zeta or Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlIT.java
+++ b/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlIT.java
@@ -74,7 +74,7 @@ import java.util.Properties;
 
 public class JdbcMysqlIT extends AbstractJdbcIT {
 
-    private static final String MYSQL_IMAGE = "mysql:latest";
+    private static final String MYSQL_IMAGE = "mysql:8.0";
     private static final String MYSQL_CONTAINER_HOST = "mysql-e2e";
     private static final String MYSQL_DATABASE = "seatunnel";
     private static final String MYSQL_SOURCE = "source";

--- a/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-4/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMySqlCreateTableIT.java
+++ b/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-4/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMySqlCreateTableIT.java
@@ -72,7 +72,7 @@ public class JdbcMySqlCreateTableIT extends TestSuiteBase implements TestResourc
     private static final String PG_GEOMETRY_JAR =
             "https://repo1.maven.org/maven2/net/postgis/postgis-geometry/2.5.1/postgis-geometry-2.5.1.jar";
 
-    private static final String MYSQL_IMAGE = "mysql:latest";
+    private static final String MYSQL_IMAGE = "mysql:8.0";
     private static final String MYSQL_CONTAINER_HOST = "mysql-e2e";
     private static final String MYSQL_DATABASE = "auto";
 

--- a/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-4/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcSqlServerCreateTableIT.java
+++ b/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-4/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcSqlServerCreateTableIT.java
@@ -128,7 +128,7 @@ public class JdbcSqlServerCreateTableIT extends TestSuiteBase implements TestRes
     private static final String PG_GEOMETRY_JAR =
             "https://repo1.maven.org/maven2/net/postgis/postgis-geometry/2.5.1/postgis-geometry-2.5.1.jar";
 
-    private static final String MYSQL_IMAGE = "mysql:latest";
+    private static final String MYSQL_IMAGE = "mysql:8.0";
     private static final String MYSQL_CONTAINER_HOST = "mysql-e2e";
     private static final String MYSQL_DATABASE = "auto";
 

--- a/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMySqlSaveModeCatalogIT.java
+++ b/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMySqlSaveModeCatalogIT.java
@@ -52,7 +52,7 @@ public class JdbcMySqlSaveModeCatalogIT extends TestSuiteBase implements TestRes
     private static final String MYSQL_DRIVER_JAR =
             "https://repo1.maven.org/maven2/com/mysql/mysql-connector-j/8.0.32/mysql-connector-j-8.0.32.jar";
 
-    private static final String MYSQL_IMAGE = "mysql:latest";
+    private static final String MYSQL_IMAGE = "mysql:8.0";
     private static final String MYSQL_CONTAINER_HOST = "mysql-e2e";
     private static final String MYSQL_DATABASE = "auto";
 

--- a/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlSaveModeHandlerIT.java
+++ b/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlSaveModeHandlerIT.java
@@ -55,7 +55,7 @@ import java.util.stream.Collectors;
 @Slf4j
 public class JdbcMysqlSaveModeHandlerIT extends AbstractJdbcIT {
 
-    private static final String MYSQL_IMAGE = "mysql:latest";
+    private static final String MYSQL_IMAGE = "mysql:8.0";
     private static final String MYSQL_CONTAINER_HOST = "mysql-e2e-2";
     private static final String MYSQL_DATABASE = "seatunnel";
     private static final String MYSQL_SOURCE = "source";

--- a/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlSplitIT.java
+++ b/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlSplitIT.java
@@ -74,7 +74,7 @@ import static org.awaitility.Awaitility.given;
 public class JdbcMysqlSplitIT extends TestSuiteBase implements TestResource {
     private static final Logger LOG = LoggerFactory.getLogger(JdbcMysqlSplitIT.class);
 
-    private static final String MYSQL_IMAGE = "mysql:latest";
+    private static final String MYSQL_IMAGE = "mysql:8.0";
     private static final String MYSQL_CONTAINER_HOST = "mysql-e2e";
     private static final String MYSQL_DATABASE = "auto";
     private static final String MYSQL_TABLE = "split_test";

